dir.props

```diff
@@ -109,7 +109,7 @@
     <NativeBinDir>$(BinDir)$(OSGroup).$(ArchGroup).$(ConfigurationGroup)/native</NativeBinDir>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetGroup)' == 'netfx' AND '$(IsTestProject)' != 'true'">
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netfx' AND '$(IsTestProject)' != 'true' AND '$(BuildNet462AndNewer)' != 'false'">
```

Comment:
I would just remove the 'IsTestProject' part of the condition as it is currently true always since it runs against src/tests.builds. Also, I would probably add a comment above the property group saying that tests will set the BuildNet462AndNewer property to false.